randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -504,10 +504,19 @@ Extra - room_id: [13]
   * Layers: default
   > Pickup (Hidden Missile Tank)
       Trivial
-  > Pickup (Missile Tank)
-      Missiles ≥ 2
   > Door to Escape Save Room
       Can Use Any Bombs
+  > Beside Other Pickup
+      Missile Data and Missiles ≥ 2 and Morph Ball
```

Comment:
This is wrong. Using a missile resource already implies some way to fire them (which right now is having missile data).

Reply:
ah I see, Sorry, I wasn't aware of this.